s4/data.py

```diff
@@ -216,145 +218,6 @@ def create_quickdraw_dataset(bsz=128):
     return trainloader, testloader, N_CLASSES, SEQ_LENGTH, IN_DIM
 
 
-# ### FSDD Sequence Modeling
```

Comment:
I think there was a merge issue. Can you pull the latest main and make sure this part gets back in?